src/inotify/event.cr

```diff
@@ -2,35 +2,35 @@ module Inotify
   struct Event
     property name, path : String
     property mask, cookie : UInt32
-    property event_type : EventType
+    property type : Type
     @is_dir : Bool
 
     def directory?
       @is_dir
     end
 
-    def initialize(@name : String, @path : String, @mask : UInt32, @cookie : UInt32, @is_dir : Bool, @event_type : EventType)
+    def initialize(@name : String, @path : String, @mask : UInt32, @cookie : UInt32, @is_dir : Bool, @type : Type)
     end
-  end
 
-  enum EventType
-    CREATE
-    MODIFY
-    MOVE
-    MOVE_SELF
-    DELETE
-    DELETE_SELF
-    UNKNOWN
+    enum Type
+      CREATE
+      MODIFY
+      MOVE
+      MOVE_SELF
+      DELETE
+      DELETE_SELF
+      UNKNOWN
 
-    def self.parse_mask(mask : UInt32) : EventType
-      event = EventType::UNKNOWN
-      event = EventType::MODIFY if LibInotify::IN_MODIFY & mask != 0
-      event = EventType::MOVE if LibInotify::IN_MOVE & mask != 0
-      event = EventType::CREATE if LibInotify::IN_CREATE & mask != 0
-      event = EventType::DELETE if LibInotify::IN_DELETE & mask != 0
-      event = EventType::DELETE_SELF if LibInotify::IN_DELETE_SELF & mask != 0
-      event = EventType::MOVE_SELF if LibInotify::IN_MOVE_SELF & mask != 0
-      event
+      def self.parse_mask(mask : UInt32) : Type
+        event = Type::UNKNOWN
+        event = Type::MODIFY if LibInotify::IN_MODIFY & mask != 0
+        event = Type::MOVE if LibInotify::IN_MOVE & mask != 0
+        event = Type::CREATE if LibInotify::IN_CREATE & mask != 0
+        event = Type::DELETE if LibInotify::IN_DELETE & mask != 0
+        event = Type::DELETE_SELF if LibInotify::IN_DELETE_SELF & mask != 0
+        event = Type::MOVE_SELF if LibInotify::IN_MOVE_SELF & mask != 0
+        event
```

Comment:
Does the `event` variable needed? Can it be replaced by directly `return`, with `Type::UNKNOWN` at the end?